app/src/components/account/EmailAuthentication.vue

```diff
@@ -1,20 +1,23 @@
 <template>
   <SessionFrame hide-back>
-    <h2 class="session-title">
-      Magic link success!
-    </h2>
-    <div class="session-main">
-      <p class="session-subtitle">
-        You're now signed in to Lunie with your email address. Head over to the
-        notifications page to see some of your recent events.
-      </p>
-      <TmBtn value="Let's Go!" centered @click.native="goToNotifications" />
-    </div>
-
     <template v-if="account.signInError || account.signInEmailError">
       <h2 class="session-title">Magic link error! 🙀</h2>
       <div class="session-main">
         <p class="session-subtitle">{{ account.signInError.message }}</p>
+        <TmBtn value="Try again" centered @click.native="goToSignInModal" />
+      </div>
```

Comment:
Can't use a router link instead to avoid code?

Reply:
can either be a router link styled like a button ... or a button with a method /shrug